internal/experimental/examples/function-listener/print-trace.go

```diff
@@ -8,30 +8,39 @@ import (
 
 	"github.com/tetratelabs/wazero"
 	"github.com/tetratelabs/wazero/api"
-	experimentalapi "github.com/tetratelabs/wazero/internal/experimental/api"
+	"github.com/tetratelabs/wazero/experimental"
+	"github.com/tetratelabs/wazero/internal/wasmdebug"
 )
 
 type stackKey struct{}
```

Comment:
ps I left this example in the really internal area as I'm not confident this is the practice we want, yet. I put a much smaller example in the listener_test which is also easier to follow for now.